instrumentation/servlet/servlet-3.0/javaagent/src/test/groovy/AbstractServlet3Test.groovy

```diff
@@ -95,4 +100,52 @@ abstract class AbstractServlet3Test<SERVER, CONTEXT> extends HttpServerTest<SERV
         break
     }
   }
+
+  def "snippet injection with ServletOutPutStream"() {
+    setup:
+    ExperimentalSnippetHolder.setSnippet("\n  <script type=\"text/javascript\"> Test </script>")
+    def request = request(HTML_SERVLET_OUTPUT_STREAM, "GET")
+    def response = client.execute(request).aggregate().join()
+
+    expect:
+    response.status().code() == HTML_SERVLET_OUTPUT_STREAM.status
+    // check response content-length header
```

Comment:
forget the reason to put this comment here

Reply:
```suggestion
```